examples/sphere/solidbody.jl

```diff
@@ -107,10 +66,131 @@ function linkfig(figpath, alt = "")
     end
 end
 
-linkfig("output/$(dirname)/error.png", "Absolute error in height")
+FT = Float64
+ne_seq = (5, 9, 17, 33)
+Δh = zeros(FT, length(ne_seq))
+L1err, L2err, Linferr = zeros(FT, length(ne_seq)),
+zeros(FT, length(ne_seq)),
+zeros(FT, length(ne_seq))
+Nq = 4
+
+# h-refinement study
+for (k, ne) in enumerate(ne_seq)
+    domain = Domains.SphereDomain(R)
+    mesh = Meshes.Mesh2D(domain, Meshes.EquiangularSphereWarp(), ne)
+    grid_topology = Topologies.Grid2DTopology(mesh)
+    quad = Spaces.Quadratures.GLL{Nq}()
+    space = Spaces.SpectralElementSpace2D(grid_topology, quad)
+
+    coords = Fields.coordinate_field(space)
+
+    Δh[k] = 2 * R / ne
+
+    h_init = map(coords) do coord
+        n_coord = Geometry.components(Geometry.Cartesian123Point(coord))
+        n_center = Geometry.components(Geometry.Cartesian123Point(center))
+        # https://en.wikipedia.org/wiki/Great-circle_distance
+        rd = R * atan(norm(n_coord × n_center), dot(n_coord, n_center))
+
+        if test_name == gaussian_test_name
+            h0 * exp(-(rd / r0)^2 / 2)
+        else # default test case, cosine bell
+            if rd < r0
+                h0 / 2 * (1 + cospi(rd / r0))
+            else
+                0.0
+            end
+        end
+    end
+
+    u = map(coords) do coord
+        ϕ = coord.lat
+        λ = coord.long
 
-#----------------lat, long plots
-Plots.png(Plots.plot(coords.lat), joinpath(path, "latitude.png"))
-linkfig("output/$(dirname)/latitude.png", "latitude")
-Plots.png(Plots.plot(coords.long), joinpath(path, "longitude.png"))
-linkfig("output/$(dirname)/longitude.png", "longitude")
+        uu = u0 * (cosd(α0) * cosd(ϕ) + sind(α0) * cosd(λ) * sind(ϕ))
+        uv = -u0 * sind(α0) * sind(λ)
+        Geometry.UVVector(uu, uv)
```

Comment:
So I think the issue we're seeing here is that this doesn't end up defining the vector correctly at the pole if `α0` is not zero.

As Paul suggested, we should probably just define it in Cartesian coordinates (we will have to add some functionality for that)

Reply:
From Paul:
> The solid body velocity field in 3D Cartesian is \vec{u} = - (\vec{Omega} \cdot \vec{x}) (\vec{Omega} x \vec{x}), where \Omega = (cos(alpha), 0, sin(alpha))